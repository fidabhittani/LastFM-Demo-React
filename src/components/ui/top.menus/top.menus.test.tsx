import React from "react";
import { shallow } from "enzyme";
import ToMenus from "./index";

/** ToMenus TEST */
describe("ToMenus Root Component Tests", () => {
  it("ToMenus Component renders correctly", () => {
    const menusWrapper = shallow(<ToMenus />);
    expect(menusWrapper).toMatchSnapshot();
  });
});
