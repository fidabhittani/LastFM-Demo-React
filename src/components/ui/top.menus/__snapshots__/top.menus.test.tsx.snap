// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToMenus Root Component Tests ToMenus Component renders correctly 1`] = `ShallowWrapper {}`;
